src/constants/languages.js

```diff
@@ -0,0 +1,7 @@
+import LANGUAGES from '@cospired/i18n-iso-languages';
+
+LANGUAGES.registerLocale(require('@cospired/i18n-iso-languages/langs/en.json'));
+
+const ALL_LANGUAGES = LANGUAGES.getNames('en');
```

Comment:
I'm okay merging this PR, but let's create a followup ticket to i18n this.

Reply:
All set: ARCH-457